data/mods/MindOverMatter/effectoncondition/eoc_mental_engineering.json

```diff
@@ -0,0 +1,5106 @@
+          "Lose your pacifistic desires to do no harm.  Even thinking about what you might do afterwards unnerves you.",
+          "Stop caring about all the little ups and downs of the Cataclysm.  You feel this is only possible due to your extensive consumption of Medical mutagen.",
+          "Let yourself be affected by all the ups and downs of the Cataclysm.  Hopefully more up than down.",
+          "Perform a disaterous mental surgery that causes you to become completely paralyzed by the dark.  Under no circumstances is this a good idea.",
+          "Perform a comprehensive and corrective surgery to your fear of the dark.  No longer will you be paralyzed by errant shadows.",
+          "Immensely inflate your ego at your attractiveness to an incredibly overwhelming extent.  Maybe you'll feel happier?",
+          "Deflate your perfectly logical, justified, and factual acknowledgement of your beauty.  How could you deprive everyone else of your looks?",
+          "Use your incredible telepathic powers to prevent yourself from tolerating untasty food.  The markings of a real apocalyptic survivor.",
+          "Rather than expanding your pallete the normal way, use your incredible telepathic powers to force yourself to be OK with untasty looking food.  At least it gets the job done.",
+          "Force yourself to appreciate the company of others by making yourself an extrovert.  Hopefully you'll be able to find people to interact with.",
+          "Stop yourself from enjoying the company of others by removing your extroversion.  Its not like there's anybody around anyways, right?",
+          "Force yourself to require the company of others by making yourself incredibly sociable.  Hopefully you have friends.",
+          "Work on youself by stopping your need for other's presence.  A workablew lifestyle turned problem which started with the death of everyone you knew and loved.",
+          "Make sure that you're happy alone by making yourself an introvert.  With near everyone dead, this mindset just got a lot more viable.",
+          "Stop yourself from enjoying being alone by removing your introversion.  You now feel positively neutral.",
+          "Force yourself to hate interacting with others.  What would have been a debilitating mentality in pre-Cataclysm life might actually help you.  What's the chance you'll actually have to talk to someone now?",
+          "Your instinctive hatred of social interaction is probably a bit much.  For all that meeting other survivors is rare you should probably be more receptive to their presence.",
+          "Embrace your inner (and with the help of mutagen, outer) cat by making sure that you hate the touch of water.  In for a penny, in for a pound.",
+          "As much as you currently hate it, you can begrugingly accept that it would be easier if you didn't feel that way.",
```

Comment:
This description doesn't mention anything about the trait it modifies(hating water) like the others do 🤔